virttest/libvirt_xml/devices/librarian.py

```diff
@@ -10,7 +10,7 @@
 # Avoid accidental names like __init__, librarian, and/or other support modules
 DEVICE_TYPES = ['disk', 'filesystem', 'controller', 'lease',
                 'hostdev', 'redirdev', 'smartcard', 'interface', 'input',
-                'hub', 'graphics', 'video', 'parallel', 'serial', 'console',
+                'hub', 'graphics', 'video', 'audio', 'parallel', 'serial', 'console',
```

Comment:
Here is how to get what else device types are missing (base on libvirt v7.3):
Execute the following code. It will get all the device types from libvirt unit test files, then check if it is missing in the **DEVICE_TYPES** of avocado-vt
```sh
cd $LIBVIRT_SRC
for j in $(for i in tests/qemuxml2argvdata/*.xml;doecho "du //devices" | xmllint --shell $i|grep '^  [a-zA-Z0-9]\+' -o;done|sort|uniq);do
  grep -q $j ../avocado-vt/virttest/libvirt_xml/devices/librarian.py
  if [ $? -ne 0 ];then
  echo $j;
  fi;
done
```
Results: 
```
audio
nvram
redirfilter
shmem
```
Please add `nvram redirfilter shmem`, too.
BTW, for `filterref` in the DEVICE_TYPES, no such a device name in libvirt.
@chunfuwen

Reply:
@qiankehan , good suggestions,
I will address nvram redirfilter shmem  on separate PRs.
This PR is first to fix gating test issue

Reply:
Yes, thanks @chunfuwen and @qiankehan, as this device is the default added one so it will affect some existing tests, for others, we have no urgency to add them, when we need to test them, then we will add them with specific requirements.

Reply:
@chunfuwen @kylazhang OK. Let's fix that in other PRs